infrastructure/utils.py

```diff
@@ -103,10 +105,15 @@ def load_csv(geography, fp):
 
 
 @transaction.atomic
-def load_file(geography, reader, financial_year):
+def load_file(geography, reader, financial_year=None):
```

Comment:
I think this is relating to quarterly updates as opposed to annual data updates (which focus on budget numbers)

Reply:
also this isn't a good solution, I don't think